michael-rubinstein/hw_07/madlibs/js/app.js

```diff
@@ -0,0 +1,30 @@
+var startupX = ['Uber', 'Google', 'Amazon', 'Apple', 'Facebook', 'Twitter'];
+var startupY = ['Slack', 'Trello', 'Tesla', 'Hyperloop', 'Harvest'];
+var arrFavorites = [];
+
+function rndElement (arr) {
+    return arr[Math.floor(Math.random() * arr.length)];
+}
+
+$(document).ready (function() {
+    var $btnCreate = $("#create");
+    var $btnSave = $("#save");
+    var $btnPrint = $("#print");
+
+    $btnCreate.on("click", function(event) {
+        event.preventDefault();
+        var mlText = `A startup that's like ${rndElement(startupX)}, but for ${rndElement(startupY)}!`
+        $('h1').replaceWith(`<h1 id="xForY">${mlText}</h1>`);
+    });
```

Comment:
Ahh, ES6 syntax. 😸 
This is a perfect use case for template literals, Michael. 👍